### PR TITLE
Fix NPE while adding "response_body_size" breadcrumb

### LINKED ISSUE #1883: NullPointerException in SentryFeignClient NPE
_Platform:_
- [x]  Java -> If yes, which Java (and sourceCompatibility/targetCompatibility) version? **11/11**
- [x]  Spring -> If yes, which version? **Spring Boot 2.4.9 / Spring 5.3.9**

_Platform installed with:_
- [x] Maven Central

The version of the SDK:
**5.5** but AFAIK that file didn't change since that time

---
I have the following issue:

```
java.lang.NullPointerException: null
	at io.sentry.openfeign.SentryFeignClient.addBreadcrumb(SentryFeignClient.java:87) ~[sentry-openfeign-5.5.0.jar!/:na]
	at io.sentry.openfeign.SentryFeignClient.execute(SentryFeignClient.java:76) ~[sentry-openfeign-5.5.0.jar!/:na]
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:119) ~[feign-core-10.12.jar!/:na]
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:89) ~[feign-core-10.12.jar!/:na]
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:100) ~[feign-core-10.12.jar!/:na]
	at com.sun.proxy.$Proxy258.accessToClients(Unknown Source) ~[na:na]
	at XXX
```
where XXX is feignClient method call

**Steps to reproduce:**
Apparently, request was not null, while the body was.
image
In feign `Response` implementation comment states that it could be null
image

### PR DESCRIPTION
**Pull request by neseleznev**
When response body is null, set size to 0
in a same fashion "request_body_size" is set.
Add test.
Add note to the changelog

Closes #1883 , see description there

## :green_heart: How did you test it?

I wrote a test.
Had no time to build the project and run the tests locally, TBH

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [x] I updated the docs if needed
- [x] No breaking changes

### CODE REVIEW

**Comment by neseleznev** [comment]
Fixed the code style with force push

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Fix: NPE while adding "response_body_size" breadcrumb, when response body is null (#1883)
```
Comment (marandaneto) [inline CHANGELOG.md:5]:
```suggestion
* Fix: NPE while adding "response_body_size" breadcrumb, when response body is null (#1884)
```

**Comment by marandaneto** [comment]
thanks for doing that @neseleznev

**Comment by marandaneto** [comment]
> Fixed the code style with force push

still fails, run `make format`

**Review by romtsn — APPROVED** [review]

**Pull request merged**